cordova-plugin-outline/README.md

```diff
@@ -44,17 +44,17 @@ We use a custom framework dependency, PacketProcessor, that implements tun2socks
 This plugin supports Android, iOS, and macOS.
 
 ### Android
-This plugin targets Android devices running Lollipop (API 21), or higher.
+This plugin targets Android devices running Lollipop (API 22), or higher.
 
 ### Apple
-This plugin targets Apple devices running iOS 9.0+ and macOS/OS X 10.11.
+This plugin targets Apple devices running iOS 11.0+ and macOS/OS X 10.11.
 
 ## JavaScript API
 
 ```ts
 
 // Represents a Shadowsocks server configuration.
-interface ShadowsocksServerConfig {
+interface ServerConfig {
```

Comment:
Did you mean to change this name as well?

Reply:
Yes, this interface was always `ServerConfig`. The documentation was not in sync.